manifests/piped/templates/rbac.yaml

```diff
@@ -33,13 +37,59 @@ metadata:
   labels:
     {{- include "piped.labels" . | nindent 4 }}
 rules:
-- apiGroups:
-  - '*'
-  resources:
-  - '*'
-  verbs:
-  - '*'
-- nonResourceURLs:
-  - '*'
-  verbs:
-  - '*'
+  {{- if .Values.rbac.rules -}}
+  {{- with .Values.rbac.rules }}
+  {{- toYaml . | nindent 2 }}
+  {{- end }}
+  {{- else }}
+  - apiGroups:
+    - '*'
+    resources:
+    - '*'
+    verbs:
+    - '*'
+  - nonResourceURLs:
+    - '*'
+    verbs:
+    - '*'
+  {{- end }}
```

Comment:
This part can be defined by a helper function and reused inside this file.

Reply:
Ref: https://github.com/pipe-cd/pipe/blob/master/manifests/piped/templates/_helpers.tpl